### ISSUE TITLE
Configure error using make distclean

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract-ocr/tesseract#916]
I was trying to make a debug build as per https://github.com/tesseract-ocr/tesseract/issues/529#issuecomment-301166173

Configure gave the following error

```
bash ./debug.sh
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
Using git revision: 4.00.00alpha-512-g6bebe71
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking for style of include used by make... GNU
checking whether make supports nested variables... yes
configure: error: source directory already configured; run "make distclean" there first
make: *** No targets specified and no makefile found.  Stop.
```

So I ran `make distclean` which ended with the following error

```
 make distclean
Making distclean in arch
make[1]: Entering directory `/mnt/c/Users/User/shree/tesseract-head/arch'
make[2]: Entering directory `/mnt/c/Users/User/shree/tesseract-head/arch'
test -z "" || rm -f
rm -rf .libs _libs
test -z "libtesseract_avx.la libtesseract_sse.la libtesseract_arch.la" || rm -f libtesseract_avx.la libtesseract_sse.la libtesseract_arch.la

...

make[1]: Leaving directory `/mnt/c/Users/User/shree/tesseract-head/api'
Making distclean in .
make[1]: Entering directory `/mnt/c/Users/User/shree/tesseract-head'
rm -rf .libs _libs
make[2]: Entering directory `/mnt/c/Users/User/shree/tesseract-head/training'
Makefile:960: ../vs2010/port/.deps/strcasestr.Plo: No such file or directory
make[2]: *** No rule to make target `../vs2010/port/.deps/strcasestr.Plo'.  Stop.
make[2]: Leaving directory `/mnt/c/Users/User/shree/tesseract-head/training'
make[1]: *** [clean-local] Error 2
make[1]: Leaving directory `/mnt/c/Users/User/shree/tesseract-head'
make: *** [distclean-recursive] Error 1
```

**Comment by Shreeshrii** [comment]
```
./configure  --enable-debug --disable-shared --disable-static CXXFLAGS="-g"
```
works fine.

The error comes when trying to build in a subdir with 

```
../../configure  --enable-debug --disable-shared --disable-static CXXFLAGS="-g"
```

**Comment by amitdo** [comment]
Was it solved?

**Comment by stweil** [comment]
No, this is still an issue. `./configure && make distclean` fails in the source root directory.

**Comment by stweil** [comment]
@Shreeshrii, now you can close this issue. It was fixed with PR #968.

**Comment by Shreeshrii** [comment]
Thanks!
